android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/AccountTypeEnum.kt

```diff
@@ -0,0 +1,77 @@
+package com.mytiki.sdk.capture.receipt.capacitor
+
+enum class AccountTypeEnum(val type: TypeEnum,  val source: String) {
```

Comment:
Since it is not a real Enum, change this to AccountTypeCommon. It will avoid confusion with TypeEnum.